colorls.rb

```diff
@@ -43,20 +88,23 @@ def init_icons
   end
 
   def chunkify
-    chunk_size = @contents.count
+    if @one_per_line
+      @contents.map { |x| [x] }
```

Comment:
Okay. I guess there will be quite a few places my code can be improved, it's my first time in ruby.